### ISSUE TITLE
[Feature Request] Delete custom domain / re-provision

### ISSUE THREAD

**Issue by simenandre** [aerobatic/aerobatic-cli#3]
I was to late to validate my domain, so the validation is timed out. Now I'm stuck not being able to delete the domain or restart the validation process.

I think there should be a delete functionality either way?

**Comment by dvonlehman** [comment]
I agree it would be a good addition to force re-provisioning a domain if the original cert failed to be validated in time.

**Comment by cobraz** [comment]
It would also be nice if you just typed in the wrong domain 👍

**Comment by cobraz** [comment]
Let me know if you want help with the cli!

**Comment by cobraz** [comment]
@dvonlehman Are there any updates on this? As previously stated, I'm happy to help! Just like to get the _go-ahead_ first :)

**Comment by dvonlehman** [comment]
@cobraz that would be fantastic. I will need to make a companion update to our API. The CLI would need to make a `DELETE` request to `/:customerSlugOrId/domains?domainName=xyz.com`.

**Comment by cobraz** [comment]
@dvonlehman Is it possible for me to get a demo/dev account to help out with the CLI?

**Comment by dvonlehman** [comment]
@cobraz are you thinking of an Aerobatic dev account? We don't have a publicly accessible test API endpoint, but you should be able to verify that the CLI is making the expected REST calls with unit tests. I can e2e test it before the pull request is merged.

**Comment by cobraz** [comment]
Okey, great :) I'll look into this in a few days :)
